src/main/java/de/wejago/hichi2influx/service/SensorMessageSubscriber.java

```diff
@@ -0,0 +1,36 @@
+package de.wejago.hichi2influx.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.wejago.hichi2influx.dto.SensorEntry;
+import de.wejago.hichi2influx.repository.InfluxDbRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.paho.client.mqttv3.IMqttMessageListener;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+    public class SensorMessageSubscriber implements IMqttMessageListener {
+        private final ObjectMapper objectMapper;
+
+        private final InfluxDbRepository influxDbRepository;
+
+        @Override public void messageArrived(String s, MqttMessage mqttMessage) throws Exception {
+            try {
+                log.info("Received message: " + new String(mqttMessage.getPayload()));
```

Comment:
Extract mqttMessage.getPayload() as var and use it at both locations, currently you create 2 strings!